src/test/java/org/broadinstitute/consent/http/mail/message/DACMembersDARRADARApprovedMessageTest.java

```diff
@@ -0,0 +1,62 @@
+package org.broadinstitute.consent.http.mail.message;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import freemarker.template.Template;
+import java.io.StringWriter;
+import java.io.Writer;
+import java.util.List;
+import org.broadinstitute.consent.http.configurations.FreeMarkerConfiguration;
+import org.broadinstitute.consent.http.mail.freemarker.FreeMarkerTemplateHelper;
+import org.broadinstitute.consent.http.models.User;
+import org.broadinstitute.consent.http.models.dto.DatasetMailDTO;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class DACMembersDARRADARApprovedMessageTest {
```

Comment:
The long sequence of all caps is challenging to read. I think what you have here is standard, but it's like reading twitter.